docs/api.rst

```diff
@@ -2,6 +2,42 @@ API Reference
 =============
 
 
+HTTP Sessions
+-------------
+
+By default, the HTTP session used for getting credentials is reused for
+API calls (recommended if there are many). If this is not desired, you
+can pass in your own ``aiohttp.ClientSession`` instance into
+``AIOGoogleDNSClient`` or ``AIOGoogleHTTPClient``. The auth client
+``GoogleAuthClient`` may also take an explicit session object, but is
+not required to assert a different HTTP session is used for the API
+calls.
+
+.. code-block:: python
+
+    import aiohttp
+
+    from gordon_janitor_gcp import auth
+    from gordon_janitor_gcp import AIOGoogleDNSClient
+    from gordon_janitor_gcp import AIOGoogleHTTPClient
+
+    keyfile = '/path/to/service_account_keyfile.json'
+    session = aiohttp.ClientSession()  # optional
+    auth_client = auth.GoogleAuthClient(
+        keyfile=keyfile, session=session)
+    creds = auth_client.creds
+
+    new_session = aiohttp.ClientSession()
+
+    # basic HTTP client
+    client = AIOGoogleHTTPClient(creds=creds, session=new_session)
```

Comment:
Could you update these args and the ones for `AIOGoogleDNSClient` right below with the updated method signatures (`creds=creds` -> `auth_client=auth_client`)? My bad, I should have pointed out the instances where `creds` are used in my previous comments.